Build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Для локального запуска на хосте, где bash живет не в /bin
```suggestion
#!/usr/bin/env bash
```